concepticondata/conceptlists/Rowe-1954-232.tsv

```diff
@@ -0,0 +1,233 @@
+Rowe-1954-232-31	31	piel	skin	763	SKIN
+Rowe-1954-232-32	32	hueso	bone	1394	BONE
+Rowe-1954-232-33	33	sangre	blood	946	BLOOD
+Rowe-1954-232-34	34	corazón	heart	1223	HEART
+Rowe-1954-232-35	35	pulmones	lungs	688	LUNG
+Rowe-1954-232-36	36	pene	penis	1222	PENIS
+Rowe-1954-232-37	37	vulva	vulva	1837	VULVA
+Rowe-1954-232-38	38	hombre	man	1554	MAN
+Rowe-1954-232-39	39	varón, macho	man	2106	MALE PERSON
+Rowe-1954-232-40	40	mujer	woman	962	WOMAN
+Rowe-1954-232-41	41	gente	people	789	PEOPLE
+Rowe-1954-232-42	42	marido	husband	1200	HUSBAND
+Rowe-1954-232-43	43	esposa	wife	1199	WIFE
+Rowe-1954-232-44	44	papá	father	1217	FATHER
+Rowe-1954-232-45	45	mamá	mother	1216	MOTHER
+Rowe-1954-232-46	46	criatura	baby, infant	1289	BABY
+Rowe-1954-232-47	47	viejo	old man	406	OLD MAN
+Rowe-1954-232-48	48	agua	water	948	WATER
+Rowe-1954-232-49	49	río	river	666	RIVER
+Rowe-1954-232-50	50	isla	island	622	ISLAND
+Rowe-1954-232-51	51	lago	lake	624	LAKE
+Rowe-1954-232-52	52	pantano	swamp	1145	SWAMP
+Rowe-1954-232-53	53	fuente; ojo de agua, manantial	spring of water	849	SPRING (OF WATER)
+Rowe-1954-232-54	54	caída de agua, catarata	waterfall	943	WATERFALL
+Rowe-1954-232-55	55	rabión, raudal	rapids	1670	RAPIDS
+Rowe-1954-232-56	56	candela, fuego	fogo		
+Rowe-1954-232-57	57	ceniza	ash	646	ASH
+Rowe-1954-232-58	58	carbón	charcoal	1188	CHARCOAL
+Rowe-1954-232-59	59	humo	smoke	778	SMOKE (EXHAUST)
+Rowe-1954-232-60	60	leña	firewood	10	FIREWOOD
+Rowe-1954-232-61	61	cielo	sky	1732	SKY
+Rowe-1954-232-62	62	lluvia	rain	658	RAIN (PRECIPITATION)
+Rowe-1954-232-63	63	viento	wind	960	WIND
+Rowe-1954-232-64	64	sol	sun	1343	SUN
+Rowe-1954-232-65	65	luna	moon	1313	MOON
+Rowe-1954-232-66	66	estrella	star	1430	STAR
+Rowe-1954-232-67	67	día	day	1225	DAY (NOT NIGHT)
+Rowe-1954-232-68	68	noche	night	1233	NIGHT
+Rowe-1954-232-69	69	trueno	thunder	1150	THUNDER
+Rowe-1954-232-70	70	relámpago	lightning	1654	LIGHTNING
+Rowe-1954-232-71	71	arco iris	rainbow	1733	RAINBOW
+Rowe-1954-232-72	72	tierra	earth	1228	EARTH (SOIL)
+Rowe-1954-232-73	73	piedra	stone	857	STONE
+Rowe-1954-232-74	74	arena	sand	671	SAND
+Rowe-1954-232-75	75	casa	home	1252	HOUSE
+Rowe-1954-232-76	76	techo	roof	769	ROOF
+Rowe-1954-232-77	77	puerta	doorway	1567	DOOR
+Rowe-1954-232-78	78	banco	seat, stool	2058	STOOL
+Rowe-1954-232-79	79	estera	mat	195	MAT
+Rowe-1954-232-80	80	hamaca, chinchorro	hammock	2019	HAMMOCK
+Rowe-1954-232-81	81	catre, cama de barbacoa	bed	1663	BED
+Rowe-1954-232-82	82	olla, paila	cooking pot	3653	COOKING POT
+Rowe-1954-232-83	83	roza, chacra, conuco	cultivated clearing	212	FIELD
+Rowe-1954-232-84	84	caserío	village	930	VILLAGE
+Rowe-1954-232-85	85	camino, sendero	path, trail	2457	PATH OR ROAD
+Rowe-1954-232-86	86	red de pescar, chinchorro	fish net	388	FISHNET
+Rowe-1954-232-87	87	anzuelo	fishhook	1074	FISHHOOK
+Rowe-1954-232-88	88	hacha	axe	677	AXE
+Rowe-1954-232-89	89	cuchillo	knife	1352	KNIFE
+Rowe-1954-232-90	90	canoa, curiara	canoe	1970	CANOE
+Rowe-1954-232-91	91	remo	paddle	3137	OAR
```

Comment:
Hmm 'remo' in Huber is mapped to PADDLE but to OAR in Morse. Maybe @tresoldi can help here?